### ISSUE TITLE
False positives in 0.16

### ISSUE THREAD

**Issue by dmitrytrager** [sds/slim-lint#88]
New slim lint does not see final new line for some reasons.
It shows messages "Use 2 (not 0) spaces for indentation." that remain unchanged with any indentation changes.

Example:
```
.row
  .columns
    h2 = title("Forgot your password?")

.row
  .medium-5.columns
    = simple_form_for resource,
      as: resource_name,
      url: password_path(resource_name),
      html: { method: :post  } do |f|

      .form-inputs
        = f.input :email, required: true
      .form-actions
        = f.button :submit, "Send me reset password instructions"

  .medium-5.medium-offset-1.columns.end
    = render "admins/shared/links"

```
No matter how I move params from line 8, it says "Use one level of indentation for parameters following the first line of a multi-line method call."

**Comment by tagliala** [comment]
Hi, I cannot replicate this:

```
$ slim-lint -V test.slim 
warning: parser/current is loading parser/ruby26, which recognizes
warning: 2.6.0-dev-compliant syntax, but you are running 2.6.0.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
slim-lint 0.16.1
slim 4.0.1
rubocop 0.62.0
ruby 2.6.0p0 (2018-12-25 revision 66547) [x86_64-darwin18]

$ slim-lint test.slim 
warning: parser/current is loading parser/ruby26, which recognizes
warning: 2.6.0-dev-compliant syntax, but you are running 2.6.0.
warning: please see https://github.com/whitequark/parser#compatibility-with-ruby-mri.
test.slim:3 [W] RuboCop: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
test.slim:10 [W] RuboCop: Layout/ExtraSpacing: Unnecessary spacing detected.
test.slim:15 [W] RuboCop: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
test.slim:18 [W] RuboCop: Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
```

Maybe it was a false positive in RuboCop?
